Source/SelfService/Web/theme/assets/logos/symbol.svg

```diff
@@ -1,15 +1,11 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 127.56 131.07">
     <defs>
         <linearGradient id="symbol-gradient" x1="0" y1="0" x2="0" y2="1">
-            <stop offset="0%" stop-color="#191A21" stop-opacity="1" />
-            <stop offset="100%" stop-color="#191A21" stop-opacity="0" />
+            <stop stop-color="#202128" stop-opacity="0.9"/>
+            <stop offset="1" stop-color="#23252E" stop-opacity="0.05"/>
```

Comment:
Reeally close now - but according to Figma, these stop opacities should be `1` and `0`
![image](https://user-images.githubusercontent.com/1014990/185656937-2aa3da17-c681-42ec-bc98-7dc36148594e.png)

```suggestion
            <stop stop-color="#202128" stop-opacity="1"/>
            <stop offset="1" stop-color="#23252E" stop-opacity="0"/>
```

Reply:
From where to you get this information? I am looking svg code and it shows opacity as 0.9 and 0.05.

<img width="1248" alt="Screenshot 2022-08-22 at 11 22 15" src="https://user-images.githubusercontent.com/19160439/185874828-429065d9-93c8-4860-a63d-05e963b33800.png">

Reply:
Okay, found it in Figma. :) There seems to be different between smaller part and bigger part. You think that we should add same gradient for both of them?

Reply:
In the SVG that is currently in the repo - the gradient applies to both of them. So there should only be a need for one :)